internal/models/task.go

```diff
@@ -11,15 +11,17 @@ type Image struct {
 }
 
 type Task struct {
-	Id           string  `json:"id,omitempty"`
-	Created      float64 `json:"created,omitempty"`
-	Updated      float64 `json:"updated,omitempty"`
-	App          string  `json:"app" binding:"required" example:"argo-watcher"`
-	Author       string  `json:"author" binding:"required" example:"John Doe"`
-	Project      string  `json:"project" binding:"required" example:"Demo"`
-	Images       []Image `json:"images" binding:"required"`
-	Status       string  `json:"status,omitempty"`
-	StatusReason string  `json:"status_reason,omitempty"`
+	Id            string  `json:"id,omitempty"`
+	Created       float64 `json:"created,omitempty"`
+	Updated       float64 `json:"updated,omitempty"`
+	App           string  `json:"app" binding:"required" example:"argo-watcher"`
+	Author        string  `json:"author" binding:"required" example:"John Doe"`
+	Project       string  `json:"project" binding:"required" example:"Demo"`
+	Images        []Image `json:"images" binding:"required"`
+	Status        string  `json:"status,omitempty"`
+	StatusReason  string  `json:"status_reason,omitempty"`
+	ProvidedToken string  `json:"token,omitempty"`
+	Validated     bool
```

Comment:
I think we should replace this with `GitCommitEnabled` or some alternative, and received it from the client.